todo/src/containers/AddTodo.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { connect } from "react-redux";
+import { addTodo } from "../components/actions";
+
+let AddTodo = ({ dispatch }) => {
+	let input;
+
+	return (
+		<div>
+			<form
+				onSubmit={e => {
+					e.preventDefault();
+					if (!input.value.trim()) {
+						return;
+					}
+					dispatch(addTodo(input.value));
+					input.value = "";
+				}}
+			>
+				<input
+					ref={node => {
+						input = node;
+					}}
+				/>
+				<button type="submit">Add Todo</button>
+			</form>
+		</div>
+	);
+};
+AddTodo = connect()(AddTodo);
+
+export default AddTodo;
```

Comment:
```suggestion
export default connect(null, {addTodo})(AddTodo);
```
Curried export statement with our action being connected in connect